proto/pbpeering/peering.go

```diff
@@ -83,3 +87,88 @@ func (p *Peering) ShouldDial() bool {
 func (x ReplicationMessage_Response_Operation) GoString() string {
 	return x.String()
 }
+
+// TODO(rb): figure how to use to use mog to maintain this mapping when states are added
```

Comment:
I've used https://github.com/reillywatson/enumcover to lint for complete enum coverage